src/main/webapp/WEB-INF/tags/instructor/feedbacks/recoveryFeedbackSessionsTable.tag

```diff
@@ -25,17 +25,17 @@
   </thead>
   <c:forEach items="${recoveryFsList.rows}" var="recoverySession" varStatus="i">
     <tr>
-      <td id="recoverycourseid${i.index + fn:length(recoveryFsList.rows)}">${recoverySession.courseId}</td>
-      <td id="recoverysessionname${i.index + fn:length(recoveryFsList.rows)}">${recoverySession.sessionName}</td>
+      <td id="recoverycourseid${i.index}">${recoverySession.courseId}</td>
```

Comment:
Can you explain this change?

Reply:
Oh this is because initially I just copied the code from `archivedCoursesTable`, but later I realized that adding the number of rows in id is just for the convenience of testing in `InstructorCoursesPageUiTest`. But for soft-deleted courses/feedback sessions, we don't need this any more since we are using different methods in `InstructorFeedbackSessionsPage` (e.g. `getRecoveryFeedbackSessionRowId`) to get the elements.

I feel that adding these new methods is necessary since soft-deleted courses/feedback sessions are from a separate panel below. This will also minimize the confusion for developers modifying this part in future :)